wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceRuntimes.java

```diff
@@ -312,6 +313,8 @@ public WorkspaceStatus getStatus(String workspaceId) {
   public CompletableFuture<Void> startAsync(
       Workspace workspace, @Nullable String envName, Map<String, String> options)
       throws ConflictException, NotFoundException, ServerException {
+    TracingTags.WORKSPACE_ID.set(workspace.getId());
```

Comment:
We could do that in the WorkspaceManger.startAsync as the first async action. WDYT?

Reply:
`WorkspaceManger.startAsync` is not traced at all. 
Since `WorkspaceManger.startAsync` does not contain any long living task - I think it's OK to keep `WorkspaceRuntimes.startAsync` traced.

Reply:
then I'll leave it as it is